cfme/configure/configuration/__init__.py

```diff
@@ -1104,7 +1095,9 @@ def delete(self, cancel=False):
         Args:
             cancel: Whether to click on the cancel button in the pop-up.
         """
-        self.delete_by_name(self.details["name"], cancel)
+        navigate_to(self, 'Delete')
+        tb.select("Configuration", "Delete this Schedule from the Database", invokes_alert=True)
```

Comment:
This is done in the 'Delete' destination, so this line is duplicate. I think this is one of the reasons we stayed away from Delete destinations.